.github/workflows/test-and-deploy.yml

```diff
@@ -17,16 +17,16 @@ jobs:
     timeout-minutes: 20
     strategy:
       matrix:
-        php: [ 7.2, 7.3, 7.4, 8.0, 8.1 ]
+        php: [ '7.1', '7.2', '7.3', '7.4', '8.0', '8.1' ]
```

Comment:
`7.1` was specifically dropped a while back since it's EOL and supporting it was blocking new development: https://github.com/twilio/twilio-php/commit/d07d96c34b5ac059b602186011e1bdca20d16155

Recommend upgrading the version of PHP you're using or using a `5.X` release of this library which supported PHP 7.1

Reply:
You never updated your composer.json though, which means you effectively did not drop support.

Reply:
`>=7.1.0` means PHP 7.1 or higher, not 7.2 or higher.

Reply:
Yup, that was a miss during the change. It should be updated to `>=7.2.0` as we no longer test against 7.1